troposphere/static/js/components/projects/detail/resources/ExternalLinkActionButtons.react.js

```diff
@@ -0,0 +1,48 @@
+define(function (require) {
+
+  var React = require('react/addons'),
+    Backbone = require('backbone'),
+    Button = require('./Button.react'),
+    actions = require('actions'),
+    modals = require('modals');
+
```

Comment:
Can we switch this to ES6 modules? Given that I'm about to PR that on Monday, it'd be good to get this converted to avoid `define(...)` hanging around.